src/metadataTextTracks.js

```diff
@@ -0,0 +1,123 @@
+import videojs from 'video.js';
```

Comment:
What do you think of renaming this file something like `adCueTextTracks.js` to make it more clear what the feature is for? Also, a comment at the top of the file describing what the feature is for would be useful, I've been adding one to each of the feature files.